test/controllers/ShutMucrControllerSpec.scala

```diff
@@ -105,17 +113,31 @@ class ShutMucrControllerSpec
 
       "return BadRequest code" in new Test {
 
-        authorizedUser()
         status(routePost(body = incorrectShutMucrJSON)) must be(BAD_REQUEST)
       }
 
       "return Shut a MUCR page" in new Test {
 
-        authorizedUser()
         val result = routePost(body = incorrectShutMucrJSON)
 
         contentAsString(result) must include(messages("shutMucr.title"))
       }
+
+      "not call SubmissionService" in new Test {
+
+        routePost(body = incorrectShutMucrJSON)
```

Comment:
For every case like here you should use `futureValue`. Line below can be invoked without waiting until this is finished

Reply:
Fixed